server/src/test/java/org/elasticsearch/common/joda/JavaJodaTimeDuellingTests.java

```diff
@@ -390,6 +391,28 @@ public void testDuelingStrictParsing() {
 
         assertSameDate("2012-W31-5", "strict_weekyear_week_day");
         assertParseException("2012-W1-1", "strict_weekyear_week_day");
+//failing below
+        assertSameDate("2015-01-04T00:00Z", "strict_date_optional_time||epoch_millis");
+    }
+//Failing
+    public void testFailingParseJavaTime(){
+        DateFormatter formatter = DateFormatters.forPattern("strict_date_optional_time||epoch_millis");
+        TemporalAccessor parse = formatter.parse("2015-01-04T00:00Z");
+    }
+    //Passing
+    public void testParsingJoda(){
```

Comment:
these two methods are not needed anymore, correct?

Reply:
these are not needed. I only left them if you wanted to verify that quickly on your own